src/i18n/locale-fr.json

```diff
@@ -928,8 +928,8 @@
     "core.errors.select.repository.warning.msg": "Cliquez sur l'un des dépôts ci-dessous pour vous y connecter",
     "core.errors.or.create.repository.warning.msg": "ou créez un nouveau dépôt",
     "core.errors.no.accessible.warning.msg": "Il n'y a pas de dépôt accessible",
-    "core.errors.writable": "accessible en écriture",
-    "core.errors.repositories": "accessible en écriture",
+    "core.errors.writable": "",
```

Comment:
This will not work like this.

As can be seen in the template this is a compound label that in english looks like this:
```
"There are no accessible" + "writable" + "repositories"
```
Where the `writable` part is optional if there is a repository but it is restricted.

This can not be translated directly in french with.
The labels need to be merged into two. One without `writable` and one with.
The template would then need to be modified with an if condition and showing the apropriate label.

Reply:
![image](https://github.com/Ontotext-AD/graphdb-workbench/assets/79477644/ff0d8ac2-ef8f-4f69-a0f2-f2dc3d162f83)

Reply:
We send the same message so you don't need to add additional word in case the repository is not write accessible

Reply:
Having an optional part `writable` makes the sentences two, which show under different conditions.

```
There are no accessible repositories
```
and
```
There are no accessible writable repositories
```

Reply:
Changed the template and labels as suggested.